### DISCUSSION TITLE
How to access Dash Panel from outside your home network

### DISCUSSION THREAD

**Discussion by bluong77** [tagyoureit/nodejs-poolController#947] [Q&A] [answered]
Hi,
I know this is a noob question but how do i access the Dash Panel from outside my home network?  Right now, I access Dash Panel or even REM with my internal IP address of my stand-alone Nixie controller and 5150 or 8080 i.e. 192.168.1.201:5150 or 192.168.1.201:8080.

I added my Raspberry Pi to Port Forwarding in my router and use my external IP address (whatismyip.com) and port number (5150/8080) but all i can see is a blank menu that sort of looks like Dash Panel (but not fully loaded).  Is there other ports i need to open for Dash Panel and REM so i can access them from outside my home network?

I haven't setup Home Assistant (truthfully, i don't even have a server and understand this yet but would eventually love to set this up) - Something to do down the road.  But a second question on this? Can my Raspberry Pi/Nixie Standalone contorller also run Home Assistant server?  Is there a guide on how to do this if that's the case?

My Setup
Raspberry Pi / Building Automation Hat / 8 Relay (Stand-alone Nixie)

**Comment by tagyoureit** [comment, marked as answer]
First, don't do this in this way.  njsPC is not secured, hardened or designed to be exposed to the internet.  You are basically inviting hackers to snoop on http (unencrypted) traffic and I can guarantee there are security exploits that would be exposed.  If you want to do this properly, look into reverse tunnels, or vpn, any other secured method of accessing your home internet. (EG https://github.com/tagyoureit/nodejs-poolController/discussions/521)

Second, when you do securely set up a remote connection, you can use the checkbox below to forward requests from dashPanel to njsPC directly.

![2024-05-09_13-31-00](https://github.com/tagyoureit/nodejs-poolController/assets/1646840/31dfe8f9-0839-4e54-9e03-de995105c38e)
  (Without the checkbox, your browser is connecting to both systems directly.)

**Reply by bluong77** [reply]
Thanks for pointing me in the right direction.  I'll look into setting up a VPN server.

**Comment by NWSAlpine** [comment]
I run [ZeroTeir](https://www.zerotier.com/) on my OPNsense router. You could also run a ZeroTier client directly on the Pi.
